pom.xml

```diff
@@ -2697,7 +2830,69 @@
         <hadoop.version>3.1.0</hadoop.version>
         <curator.version>2.12.0</curator.version>
         <zookeeper.version>3.4.9</zookeeper.version>
+        <hive.group>org.apache.hive</hive.group>
+        <hive.classifier>core</hive.classifier>
+        <hive.version>2.3.4</hive.version>
+        <hive.version.short>${hive.version}</hive.version.short>
+        <hive.parquet.version>${parquet.version}</hive.parquet.version>
+        <orc.classifier></orc.classifier>
+        <calcite-avatica.artifactId>calcite-druid</calcite-avatica.artifactId>
+        <calcite.version>1.10.0</calcite.version>
+        <hive.parquet.group>org.apache.parquet</hive.parquet.group>
```

Comment:
Change `com.twitter:parquet-hadoop-bundle` to `org.apache.parquet:parquet-hadoop-bundle` from Hive 1.3.0:
https://github.com/apache/hive/commit/11020ae562aba4c4097556f2e06c6783d211c041#diff-a48f566bb7d16f7bb4eae4fabac11468L84
https://issues.apache.org/jira/browse/HIVE-11096